README.md

```diff
@@ -21,12 +21,12 @@ A simple, free software weather app for Android. It gathers data from [OpenWeath
 [<img src="https://i.imgur.com/olkk2cB.png" width=250>](https://i.imgur.com/olkk2cB.png)
 
 ## Contribution
-If you have any ideas or issues, don't hesitate to make contact via the Issues page. Every contribution is welcome. See [Contributing](https://github.com/martykan/forecastie/blob/master/CONTRIBUTING.md) for further information; please also read our [Code of conduct](https://github.com/martykan/forecastie/blob/master/CODE-OF-CONDUCT.md).
+If you have any ideas or issues, don't hesitate to make contact via the Issues page. Every contribution is welcome. See [Contributing](https://github.com/martykan/forecastie/blob/master/CONTRIBUTING.md) for further info.
 
 ## Licence
 [![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](http://www.gnu.org/licenses/gpl-3.0.en.html)  
 
-This application is Free Software: You can use, study share and improve it at your
+This app is libre software: You can use, study share and improve it at your
 will. Specifically you can redistribute and/or modify it under the terms of the
```

Comment:
I believe this was taken from gnu.org, as part of "how to apply the licence", so we should probably leave it as-is.